keps/sig-architecture/2019-03-19-go-modules.md

```diff
@@ -166,9 +213,14 @@ This section is in progress, pending investigation into necessity and feasibilit
 
 ### Graduation Criteria
 
-* kubernetes/kubernetes vendor management uses go modules
-* published staging components can be successfully consumed by go module consumers
-* published staging components can be successfully consumed by GOPATH-based consumers
+* `k8s.io/kubernetes` vendor management uses go modules
+* there are documented processes for:
```

Comment:
Will you be doing this in a feature branch? It seems like something that might work sort of well that way.

Reply:
probably not beyond the normal feature branch per PR... the k/k changes are all in https://github.com/kubernetes/kubernetes/pull/74877, and I don't plan to merge that until these items are complete

@sttts also has the publishing bot running off of my branch to a test org in https://github.com/kubernetes/publishing-bot/pull/169 (https://github.com/kubernetes/publishing-bot/pull/169/commits/c3d0bbbfc9e4d6161a87ccaff750c7b1652977a0) so we can also verify the publishing bits work as intended pre-merge